### PR TITLE
upgrade-operator: Improve helm instructions and add tanzu upgrade

### PR DESCRIPTION
**Pull request by ssapra**
Authored-by: Shaan Sapra <shsapra@vmware.com>

Name the branches to merge this change with or enter "None". 1.6

### CODE REVIEW

File: upgrade-operator.html.md.erb [hunk 1]
```diff
@@ -8,105 +8,233 @@ This topic describes how to upgrade the <%= vars.product_full %> Operator.
...
+    Upgrade using:
 
-    ``` 
-    helm ls -n tanzu-mysql-for-kubernetes-system
     ```
+    helm upgrade <RELEASE-NAME> ${CHART_DIR} -f /<path-to-your-file>/operator_overrides_values.yaml --wait
+    ```
+    where 
+     - `RELEASE-NAME` is the name of the helm release listed from the list output
+     - `operator_overrides_values.yaml` is the overrides file
```
Comment (lenapivotal) [inline upgrade-operator.html.md.erb:73]: @ssapra is the `RELEASE-NAME` the value under the `NAME` column in the `helm ls` command?

Comment (ssapra) [reply upgrade-operator.html.md.erb:73]: Yes, that's right.

File: upgrade-operator.html.md.erb [hunk 2]
```diff
@@ -8,105 +8,233 @@ This topic describes how to upgrade the <%= vars.product_full %> Operator.
...
+
+
+### <a id="upgrading_operator"></a>Upgrading the Operator
 
-    If you skip this step, the following error message might appear:
-    ``` Error: this feature has been marked as experimental and is not enabled by default.``` 
+The following steps assume that you have followed one of the two methods above to download the Helm chart.
 
-1. Use Helm to log in to the Tanzu Network Registry by running:
+1. Set the environment variable `CHART_DIR` to the location of the downloaded helm chart.
```
Comment (lenapivotal) [inline upgrade-operator.html.md.erb:32]: @ssapra Does this apply to customers that are upgrading using a downloaded file and a private registry?

File: upgrade-operator.html.md.erb [hunk 3]
```diff
@@ -8,105 +8,233 @@ This topic describes how to upgrade the <%= vars.product_full %> Operator.
...
+    where `MYSQL-OPERATOR-NAMESPACE` is the namespace.
 
+    List the helm releases using:
+    
     ```
-    helm ls -n tanzu-mysql-for-kubernetes-system
+    helm ls
     ```
-    For example:
+    ```
```
Comment (lenapivotal) [inline upgrade-operator.html.md.erb:61]: @ssapra if the customer downloaded the new operator, and updated their private registry with the new operator image, does this `helm ls` command show them the new operator images?

File: upgrade-operator.html.md.erb (hunk 2 above)
Comment (ssapra) [reply upgrade-operator.html.md.erb:32]: In either helm flow, the access resources methods wants the user to download the helm chart (either with `helm pull` or logging into TanzuNet and downloading the tarball). That's why we try to make it clear that the steps like Apply the new CRDs and run `helm upgrade ...` are using the downloaded chart.

Nothing to do with private registry.

File: upgrade-operator.html.md.erb [hunk 4]
```diff
@@ -8,105 +8,233 @@ This topic describes how to upgrade the <%= vars.product_full %> Operator.
...
     ```
-    helm chart pull registry.tanzu.vmware.com/tanzu-mysql-for-kubernetes/tanzu-mysql-operator-chart:1.6.1
+    tanzu package repository list -A
     ```
     ```
-    helm chart export registry.tanzu.vmware.com/tanzu-mysql-for-kubernetes/tanzu-mysql-operator-chart:1.6.1 -d ${CHART_DIR}
+    / Retrieving repositories...
+    NAME                 REPOSITORY                                                                          TAG    STATUS               DETAILS  NAMESPACE
+    tds-repo             registry.tanzu.vmware.com/packages-for-vmware-tanzu-data-services/tds-packages      1.3.0  Reconcile succeeded           data-services-operators
+    tanzu-rabbitmq-repo  registry.tanzu.vmware.com/p-rabbitmq-for-kubernetes/tanzu-rabbitmq-package-repo     1.3.0  Reconcile succeeded           rabbitmq-operator
```
Comment (lenapivotal) [inline upgrade-operator.html.md.erb:114]: @ssapra would this show 1.4.0 or 1.3.0? We shipped 1.6.1 with TDS 1.4.0

File: upgrade-operator.html.md.erb [hunk 5]
```diff
@@ -8,105 +8,233 @@ This topic describes how to upgrade the <%= vars.product_full %> Operator.
...
 
-    <p class="note">
-      <strong>Note:</strong> You can ignore the warnings in the output.
-    </p>
+    ```
+    / Retrieving package versions for mysql-operator.with.sql.tanzu.vmware.com...
+    NAME                                      VERSION  RELEASED-AT
+    mysql-operator.with.sql.tanzu.vmware.com  1.5.0    2022-07-07 17:26:16 -0700 PDT
+    mysql-operator.with.sql.tanzu.vmware.com  1.6.0    2022-09-16 12:59:05 -0700 PDT
+    mysql-operator.with.sql.tanzu.vmware.com  1.6.1    2022-09-29 09:49:15 -0700 PDT
```
Comment (lenapivotal) [inline upgrade-operator.html.md.erb:174]: @ssapra would 1.6.0 show up? I thought we pulled it out...

Comment (ssapra) [reply upgrade-operator.html.md.erb]: Yes, it is technically still in this output despite the retraction. For now, let's leave it and we may retract it later. Sorry for the confusion.

**Pull request merged**